single_tech_samples/synapseanalytics/sample2_synapseserverless/scripts/config/data-retention-config.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Can you please rename the file to datalake_config.json and replace with the file I've pasted in the teams chat this morning? The last proposal is also available here: https://github.com/orgs/Azure-Samples/projects/15/views/1?pane=issue&itemId=17285672

Reply:
changed accordingly, resolving conversation